messenger.rst

```diff
@@ -2462,6 +2466,7 @@ may want to use:
                         <framework:middleware id="doctrine_transaction"/>
                         <framework:middleware id="doctrine_ping_connection"/>
                         <framework:middleware id="doctrine_close_connection"/>
+                        <framework:middleware id="doctrine_open_transaction_logger"/>
```

Comment:
I didn't found these name in codebase. I think is generated from classname but I'm not 100% sure